src/nvim/memory.h

```diff
@@ -69,4 +69,9 @@ typedef struct {
     (void)(*ptr_); \
   } while (0)
 
+#define XFREE_CLEARN(ptr) \
```

Comment:
can just use `XFREE_CLEAR` directly. freeing a null-pointer is a no-op.

Reply:
I was wondering that, then I'll just to that.